### ISSUE TITLE
Wrong number of row in to_latex output for MultiIndex Dataframe

### ISSUE THREAD

**Issue by frubino** [pydata/pandas#9778]
If I try to run this code:

```python

pd.DataFrame.from_dict({
    ('c1', 0): pd.Series({x: x for x in range(4)}),
    ('c1', 1): pd.Series({x: x+4 for x in range(4)}),    
    ('c2', 0): pd.Series({x: x for x in range(4)}),
    ('c2', 1): pd.Series({x: x+4 for x in range(4)}),
    ('c3', 0): pd.Series({x: x for x in range(4)}),
}).T.to_latex()
```

The result is 2 rows instead of 5 in the latex output. If I use a csv it works.

I'm running pandas 0.16.0 under python 2.7 on Mac OS X 10.10

**Comment by jreback** [comment]
will mark as a bug. mi index handling was added in #7982.

**Comment by JohnSmizz** [comment]
Odd thing is it worked fine in last pandas I was using, it just stopped working properly in most recent update. My MI is in the rows, and the LaTeX output seems very confused -- some rows omitted and the data for the row is not correct either -- check eg results to this example call on pd 0.15.2

````
index = pd.MultiIndex.from_product([np.arange(2), np.arange(2)])
df = pd.DataFrame(index=index, data=np.random.rand(4,2))
````
df.head() <> df.to_latex() for both multiindex as well as the actual data...

**Comment by shoyer** [comment]
Fixed by #9908

**Comment by frubino** [comment]
Thanks
